src/components/sbb-tab-title/sbb-tab-title.scss

```diff
@@ -0,0 +1,118 @@
+// Default component properties, defined for :host. Properties which can not
+// travel the shadow boundary are defined through this mixin
+@include host-component-properties;
+
+:host {
+  display: inline-block;
+  max-width: 100%;
+  -webkit-tap-highlight-color: transparent;
+}
+
+.sbb-tab-title {
+  position: relative;
+  margin: 0;
+  min-height: var(--sbb-spacing-fixed-10x);
+  display: flex;
+  align-items: center;
+  padding-block: var(--sbb-spacing-fixed-2x);
+  padding-inline: var(--sbb-spacing-fixed-5x);
+  gap: var(--sbb-spacing-fixed-1x);
+  user-select: none;
+  cursor: pointer;
+  transition-duration: var(--sbb-animation-duration-4x);
+  transition-timing-function: var(--sbb-animation-easing);
+  transition-property: color;
+
+  &::before {
+    position: absolute;
+    content: '';
+    inset: 0;
+    border: var(--sbb-border-width-1x) solid var(--sbb-color-cloud-default);
+    border-radius: var(--sbb-spacing-fixed-6x);
+    background-color: var(--sbb-color-white-default);
+    transition-duration: var(--sbb-animation-duration-4x);
+    transition-timing-function: var(--sbb-animation-easing);
+    transition-property: inset, background-color, border-color, box-shadow;
+  }
+
+  .sbb-tab-title-text {
+    @include text-xs--bold;
+    @include fontSmoothing;
+    @include ellipsis;
+  }
+
+  ::slotted(*),
```

Comment:
If we would add a span around the content, we could address only this span instead of every content element. Maybe this could have a positive effect on the transition which then is done only on one element. What do you think?

Reply:
I believe I have discussed this possibility before with Lukas, but we preferred not to have inner wrapping elements within the heading tags. But if you want we can check with him again.